app/javascript/packages/verify-flow/steps/personal-key/download-button.tsx

```diff
@@ -29,7 +29,9 @@ function DownloadButton({ content, fileName, ...buttonProps }: DownloadButtonPro
     if (hasProprietarySaveBlob(window.navigator)) {
       event.preventDefault();
       const blob = new Blob([content], { type: 'text/plain' });
-      window.navigator.msSaveBlob(blob, fileName);
+      if (window.navigator.msSaveBlob) {
+        window.navigator.msSaveBlob(blob, fileName);
+      }
```

Comment:
Per prior discussion, we don't need this change anymore, do we?

```suggestion
      window.navigator.msSaveBlob(blob, fileName);
```